UnitySetup/UnitySetup.psm1

```diff
@@ -1870,7 +1870,36 @@ function Start-UnityEditor {
             $process.Start() | Out-Null
 
             if ( $Wait ) {
-                $process.WaitForExit()
+                if( $LogFile ) {
+                    $ljob = Start-Job -ScriptBlock {
+                        param($log)
+
+                        while ( -not (Test-Path $log -Type Leaf) ) {
+                            Start-Sleep -Milliseconds 100
+                        }
+
+                        Get-Content "$log" -Wait
+                    } -ArgumentList $LogFile
+
+                    while ( -not $process.HasExited )
+                    {
+                        do {
+                            $logOutput = Receive-Job $ljob
```

Comment:
@jwittner this line is invalid. Doesn't work for me, is there another way to write it?